Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
+    shutup do
```

Comment:
`shutup` is unneeded here; please only use it when there's output that needs suppressed.